app/src/main/java/com/gathering/android/common/Composables.kt

```diff
@@ -231,5 +234,23 @@ fun ProgressBar(
     }
 }
 
+@Composable
+fun NavigationBarPaddingSpacer() {
+    Spacer(
+        modifier = Modifier
+            .height(50.dp)
+            .navigationBarsPadding()
+    )
+}
 
+@Composable
+fun ShowText(
```

Comment:
not a good name. Composable names should not be starting with verbs. the name should be noun.